assets/src/edit-story/elements/text/util.js

```diff
@@ -44,3 +44,11 @@ export const getHandleKeyCommand = ( setEditorState ) => ( command, currentEdito
 	}
 	return 'not-handled';
 };
+
+export const generateFontFamily = ( fontFamily, fontFallback ) => {
+	let fontFamilyDisplay = fontFamily ? `${ fontFamily }` : null;
+	if ( fontFallback && fontFallback.length ) {
+		fontFamilyDisplay += `,${ fontFallback.join( `,` ) }`;
```

Comment:
Looks like a good utility function to add 1-2 tests for.

If `fontFamily` is `null` and `fontFallback` is `[ 'Arial', 'Helvetica' ]`, this function will return `null,Arial,Helvetica`.

Reply:
When would this happen? Font family and font fallback are set the same time.

Reply:
When someone accidentally passes a falsy value for `fontFamily`? Might not happen in our controlled environment, but I was just looking at this function in isolation.

Reply:
Any recommendations on what the logic should look like?

Reply:
Not having that leading comma when `fontFamilyDisplay` is `null`.

Reply:
Does this work for you @swissspidy - https://github.com/ampproject/amp-wp/pull/3998/commits/c326c9ae1bc59423e53f3a9fb3c6a56fc12e7570

Reply:
Yeah it does 👍